cypress/utils/filters.js

```diff
@@ -75,21 +78,37 @@ function applyFilters(filters, filtersConf) {
   }
 }
 
-function urlParamConvert(key, value) {
-  const filterCategory = _.find(
-    _.values(FILTER_CATEGORIES),
-    (it) => it.urlParam === key
-  );
-  const title = _.capitalize(filterCategory.title);
-  const label = _.find(filterCategory.values, (it) => it.value === value).label
-    .props.children;
-  return [title, label];
+function urlParamConvert(key, value, type) {
+  if (type === 'recommendation') {
+    const filterCategory = _.find(
+      _.values(FILTER_CATEGORIES),
+      (it) => it.urlParam === key
+    );
+    const title = _.capitalize(filterCategory.title);
+    const label = _.find(filterCategory.values, (it) => it.value === value)
+      .label.props.children;
+    return [title, label];
+  } else if (type === 'cluster') {
+    const filterCategory = _.find(
+      _.values(CLUSTER_FILTER_CATEGORIES),
+      (it) => it.urlParam === 'hits'
+    );
+    const title = _.capitalize(filterCategory.title);
+    const label = _.find(filterCategory.values, (it) => it.value === value)
+      ?.label.props.children;
+    console.log(title);
+    console.log(label);
+    return [title, label];
+  }
 }
 
-function hasChip(name, value) {
-  cy.contains(CHIP_GROUP, name).parent().contains(CHIP, value);
+function hasChip(name, value, type) {
+  if (type === 'recommendation') {
```

Comment:
what is the difference at each branch of the if?

Reply:
\+, the two branches are identical